clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -7519,7 +7523,10 @@ void ItaniumMangleContextImpl::mangleCXXCtorVTable(const CXXRecordDecl *RD,
   // <special-name> ::= TC <type> <offset number> _ <base type>
   CXXNameMangler Mangler(*this, Out);
   Mangler.getStream() << "_ZTC";
-  Mangler.mangleCXXRecordDecl(RD);
+  bool CompatibilityForV19 =
+      getASTContext().getLangOpts().getClangABICompat() <=
+      LangOptions::ClangABI::Ver19;
```

Comment:
This seems like the place to add a comment, something like "Older versions of clang did not add the record as a substitution candidate here." And the variable name should be something like `SuppressSubstitution`, or just do `/* suppress substitution */ getASTContext()...` as the argument.